synapse/handlers/pagination.py

```diff
@@ -130,10 +132,19 @@ def purge_history_for_rooms_in_range(self, min_ms, max_ms):
         else:
             include_null = False
 
+        logger.info(
+            "[purge] Running purge job for range ] %d ; %d ] (include NULLs = %s",
```

Comment:
is one of these `]`s backwards deliberately?

also, dangling `(`

Reply:
> is one of these `]`s backwards deliberately?
>

Yep: https://en.wikipedia.org/wiki/Interval_(mathematics)#Including_or_excluding_endpoints

(it's just the notation for intervals that I've always used/seen used)

> also, dangling `(`

Good catch, thanks, fixed in 4087427f0

Reply:
oh, odd, I've never seen that before; the `(a,b]` is much more familiar.

Reply:
I guess it's a cultural thing, I had never seen the `(a,b]` one before you mentioned it on a review a few months ago.